app/templates/imageresults.html

```diff
@@ -0,0 +1,119 @@
+  </style>
+  <div class="n692Zd">
+   <div class="BnJWBc">
+    <a class="lXLRf" href="/?safe=off&amp;gbv=1&amp;output=images&amp;ie=UTF-8&amp;tbm=isch&amp;sa=X&amp;ved=0ahUKEwjhh7TZyd_vAhWShf0HHeYzCmsQPAgC">
+     <img alt="Google" class="kgJEQe" src="/images/branding/searchlogo/1x/googlelogo_desk_heirloom_color_150x55dp.gif"/>
+    </a>
+   </div>
+   <div class="FbhRzb">
+    <form action="/search">
+     <input name="safe" type="hidden" value="off"/>
+     <input name="gbv" type="hidden" value="1"/>
+     <input name="ie" type="hidden" value="ISO-8859-1"/>
+     <input name="tbm" type="hidden" value="isch"/>
+     <input name="oq" type="hidden"/>
+     <input name="aqs" type="hidden"/>
+     <table class="cvifge">
+      <tr>
+       <td class="O4cRJf">
+         <!-- search input -->
+       </td>
+       <td class="O1ePr">
+        <input class="xB0fq" type="submit" value="Suche"/>
+       </td>
+      </tr>
+     </table>
+    </form>
+   </div>
+   <div class="M7pB2">
+    <!-- search options -->
+   </div>
+  </div>
+  <!-- <div class="X6ZCif"> Not present in mobile
+  </div> -->
+  <div>
+    <div>
+      <div>
+       <div class="lIMUZd">
+        <table class="By0U9">
+          <!-- correction suggested -->
+        </table>
+       </div>
+      </div>
+    </div>
+    <table class="GpQGbf">
+      {% for i in range((length // 4) + 1) %}
+      <tr>
+        {% for j in range([length - (i*4), 4]|min) %}
+          <td align="center" class="e3goi">
+          <div class="svla5d">
+            <div>
+            <div class="lIMUZd">
+              <div>
+              <table class="TxbwNb">
+                <tr>
+                <td>
+                  <a href="{{ results[(i*4)+j].webpage }}">
+                  <div class="RAyV4b">
+                    <img alt="" class="t0fcAb" src="{{ results[(i*4)+j].img_tbn }}"/>
+                  </div>
+                  </a>
+                </td>
+                </tr>
+                <tr>
+                <td>
+                  <a href="{{ results[(i*4)+j].webpage }}">
+                  <div class="Tor4Ec">
+                    <span class="qXLe6d x3G5ab">
+                    <span class="fYyStc">
+                      {{ results[(i*4)+j].domain }}
+                    </span>
+                    </span>
+                  </div>
+                  </a>
+                  <a href="{{ results[(i*4)+j].img_url }}">
+                    <div class="Tor4Ec">
+                      <span class="qXLe6d F9iS2e">
+                        <span class="fYyStc">
+                          View image
```

Comment:
Could you pass this in as a template string? Something like `view_label="View Image"` and then adding here as `{{ view_label }}`. It'd be nice to reduce the number of hardcoded strings in the templates in order to better support localization in the future.

Reply:
Yes will push the fix